core/templates/dev/head/services/AssetsBackendApiService.js

```diff
@@ -241,6 +244,11 @@ oppia.factory('AssetsBackendApiService', [
         return { audio: _audioFilesCurrentlyBeingRequested,
           image: _imageFilesCurrentlyBeingRequested
         };
+      },
+      getImageUrlForPreview: function(explorationId, filename) {
+        var bucketName = GLOBALS.GCS_RESOURCE_BUCKET_NAME;
+        return ('https://storage.googleapis.com/' + bucketName + '/' +
+          explorationId + '/assets/image/' + filename);
```

Comment:
What about dev mode? Probably use IMAGE_DOWNLOAD_URL_TEMPLATE here and interpolate the values.

Reply:
Done!